docs/api/1.0/index.rst

```diff
@@ -0,0 +1,96 @@
+| `DELETE` | :ref:`/buckets/(bucket_id)/collections/(collection_id) <collection-delete>`                  | :ref:`Delete a collection <collection-delete>`          |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| **Records**                                                                                                                                                       |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `POST`   | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records <records-post>`               | :ref:`Upload a record <records-post>`                   |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `GET`    | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records <records-get>`                | :ref:`Retrieve stored records <records-get>`            |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `PUT`    | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records/(record_id) <record-put>`     | :ref:`Replace a record <record-put>`                    |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `PATCH`  | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records/(record_id) <record-patch>`   | :ref:`Update a record <record-patch>`                   |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `GET`    | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records/(record_id) <records-get>`    | :ref:`Retrieve a single record <records-get>`           |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `DELETE` | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records/(record_id) <record-delete>`  | :ref:`Delete a single record <record-delete>`           |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+
+Buckets, Groups, Collection and Record endpoints are *resource endpoints* which
+can be filtered, paginated, and interacted with as described in
+:ref:`resource-endpoints`.
+
+
+Full reference
+==============
+
+Full detailed API documentation:
+
+.. toctree::
+   :maxdepth: 2
+
+   cliquet/utilities
+   cliquet/versioning
```

Comment:
Careful with this then, there is now two documents about versioning. I suggest you get rid of this one.